### PR TITLE
Set installed traitlets of pseudo/qe install widget when it's done

### LINKED ISSUE #261: Critical: the SSSP library is not installed
After the PR https://github.com/aiidalab/aiidalab-qe/pull/251 there is a issue that `SsspInstallWidget` can not update the state to `installed` even the pseudo libraries are properly installed. 
The reason is that `SSSPInstallWidget` once updates the `installed` traitlets in `_refresh_installed` but not in the new implementation. 
I think the way to fix this is to return the installed state from function `install()` and update the traitlets.

### PR DESCRIPTION
**Pull request by unkcpz**
fixes #261

### CODE REVIEW

File: aiidalab_qe/setup_codes.py [hunk 1]
```diff
@@ -219,6 +219,7 @@ def _refresh_installed(self):
             self.set_trait("error", str(error))
             FN_DO_NOT_SETUP.touch()
         else:
+            self.set_trait("installed", True)
```
Comment (csadorf) [inline aiidalab_qe/setup_codes.py:222]: This isn't necessary since the trait is already set on line 215.

File: aiidalab_qe/sssp.py [hunk 2]
```diff
@@ -102,6 +102,7 @@ def _refresh_installed(self):
             self.set_trait("error", str(error))
             self.set_message(str(error))
         else:
+            self.set_trait("installed", True)
```
Comment (csadorf) [inline aiidalab_qe/sssp.py:105]: To be safe I'd recommend:
```suggestion
            self.set_trait("installed", not bool(pseudos_to_install()))
```

**Review by csadorf — CHANGES_REQUESTED** [review]

File: aiidalab_qe/setup_codes.py (hunk 1 above)
Comment (unkcpz) [reply aiidalab_qe/setup_codes.py:220]: That's true, didn't notice that. I remove line 215 to align with the `install()` of `sssp.py`

File: aiidalab_qe/sssp.py (hunk 2 above)
Comment (unkcpz) [reply aiidalab_qe/sssp.py]: Good advice! If all the libraries are install by hands `bool(pseudos_to_install())` will be `False` empty list.

**Review by csadorf — APPROVED** [review]

**Pull request merged**